facebook/ThreatExchange
# Make fetch pull from te and save locally

Summary
---------

Make a fetch command that reads API tokens and collab configs from TE, fetches from ThreatExchange, and then stores the result locally.

Also add a type argument to the aws command threatupdates command. To push these changes to pypi we bump the version

A future PR will have this write the result to AWS. Good amount of hard coded stuff in here now that will be removed such as table names and secret keys

Test Plan
---------
1. upload collab configs to DynamoDB
2. upload TE app token to AWS Secrets Manager
3. Before the python-threatexchange version bump we have to direct python to our local copy of python-threatexchange with:
```python3 -m pip install python-threatexchange/```
4. Run fetcher:
```
python3 -m hmalib.lambdas.fetcher
```
5. See if the correct collab data is read correctly by looking for subdirectories under `temp_te_data` directory
